maven/README.md

```diff
@@ -17,7 +17,7 @@ kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/maven
 - **PROXY_USER**: Username to login to the proxy server (to be inserted into ~/.m2/settings.xml)
 - **PROXY_PASSWORD**: Password to login to the proxy server (to be inserted into ~/.m2/settings.xml)
 - **PROXY_HOST**: Hostname of the proxy server (to be inserted into ~/.m2/settings.xml)
-- **NON_PROXY_HOST**: Non proxy hosts to be reached directly bypassing the proxy (to be inserted into ~/.m2/settings.xml)
+- **HTTP_PROXY_NONPROXYHOSTS**: Non proxy hosts to be reached directly bypassing the proxy (to be inserted into ~/.m2/settings.xml)
```

Comment:
A particular reason why to not snake case the last part like in `HTTP_PROXY_NON_PROXY_HOSTS`.

Also as all other env vars start with `PROXY_` why not stick to the scheme ? My suggestion would `PROXY_NON_PROXY_HOSTS`